lib/Sema/CSSimplify.cpp

```diff
@@ -664,10 +666,10 @@ matchCallArguments(ConstraintSystem &cs, ConstraintKind kind,
       if (tupleArgType->getNumElements() != 1 ||
           (!cs.getASTContext().isSwiftVersion3() &&
            tupleArgType->getElement(0).hasName())) {
-        return ConstraintSystem::SolutionKind::Error;
+        return ConstraintSystem::TypeMatchResult::failure(cs, locator);
       }
     }
-    return ConstraintSystem::SolutionKind::Solved;
+    return ConstraintSystem::TypeMatchResult::success(cs);
```

Comment:
This could be cs.getTypeMatchFailure(), right?

Reply:
Same with some of the other places above and below.

Reply:
Hm, I thought I auto replaced everything, but looks like not :(